.envrc

```diff
@@ -37,7 +37,8 @@ report_to_sentry() {
     if ! require sentry-cli; then
         curl -sL https://sentry.io/get-cli/ | bash
     fi
-    SENTRY_DSN=${SENTRY_DEVENV_DSN} \
+    # Report to sentry-dev-env
+    SENTRY_DSN="https://9bdb053cb8274ea69231834d1edeec4c@o1.ingest.sentry.io/5723503" \
```

Comment:
This DSN is the new project called `sentry-dev-env` (since the original one has been renamed).